training/trainer.py

```diff
@@ -12,12 +12,10 @@
 from peft import (LoraConfig, get_peft_model)
 from torch.utils.data import Dataset
 from transformers import LlamaForCausalLM, Trainer, TrainingArguments, AutoConfig
```

Comment:
honestly we should burn this code path down - worth a separate PR to purge dead code/files soon.